cfme/v2v/migration_plans.py

```diff
@@ -0,0 +1,636 @@
+                logger.info("{} not present in filter dropdown!".format(type))
+            self.search_box.fill(name)
+            self.after_fill(was_change=True)
+
+        def select_by_name(self, vm_name):
+            self.filter("VM Name", vm_name)
+            vms_selected = []
+            for row in self.table.rows():
+                if vm_name in row.read()["VM Name"]:
+                    row.select.fill(True)
+                    vms_selected.append(row.read()["VM Name"])
+            return vms_selected
+
+    @View.nested
+    class instance_properties(View):  # noqa
+        table = Table('.//div[contains(@class, "container-fluid")]/table')
+        edit_instance_properties = Text(locator='.//button/span[contains(@class, "pficon-edit")]')
+        select_security_group = Select(locator='.//td[5]/select')
+        select_flavor = Select(locator='.//td[6]/select')
+        save_properties = Text(locator='.//div[contains(@class, "inline-edit-buttons")]'
+                                       '/button[contains(@aria-label, "Save")]')
+
+        @property
+        def is_displayed(self):
+            return (self.table.is_displayed and
+                    (len(self.browser.elements(".//div[contains(@class,'spinner')]")) == 0))
+
+        def fill(self, values):
+            osp_security_group = values.get('osp_security_group')
+            osp_flavor = values.get('osp_flavor')
+            if osp_security_group or osp_flavor:
+                self.edit_instance_properties.click()
+                self.select_security_group.wait_displayed()
+                if osp_security_group:
+                    self.select_security_group.fill(osp_security_group)
+                if osp_flavor:
+                    self.select_flavor.fill(osp_flavor)
+                self.save_properties.click()
+            self.after_fill(was_change=True)
+
+        def after_fill(self, was_change):
+            self.parent.next_btn.click()
+
+    @View.nested
+    class advanced(View):  # noqa
+        pre_playbook = BootstrapSelect("preMigrationPlaybook")
+        post_playbook = BootstrapSelect("postMigrationPlaybook")
+        pre_checkbox = Text(locator='.//input[contains(@id, "pre_migration_select_all")]')
+        post_checkbox = Text(locator='.//input[contains(@id, "post_migration_select_all")]')
+
+        @property
+        def is_displayed(self):
+            return self.pre_playbook.is_displayed
+
+        def fill(self, values):
+            self.pre_playbook.wait_displayed("5s")
+            pre_playbook = values.get('pre_playbook')
+            post_playbook = values.get('post_playbook')
+            if pre_playbook:
+                self.pre_playbook.fill(pre_playbook)
+                self.pre_checkbox.click()
+            if post_playbook:
+                self.post_playbook.wait_displayed("5s")
+                self.post_playbook.fill(post_playbook)
+                self.post_checkbox.click()
+            self.after_fill(was_change=True)
```

Comment:
These fill methods should be returning a boolean.

https://github.com/RedHatQE/widgetastic.core/blob/master/src/widgetastic/widget/base.py#L519